src/app/public/modules/selection-box/selection-box.component.ts

```diff
@@ -138,6 +139,11 @@ export class SkySelectionBoxComponent implements AfterViewInit, OnDestroy {
   }
 
   private updateDisabledState(): void {
-    this.disabled = !!this.control?.disabled;
+    this.control.disabledChange
```

Comment:
Should we change this to `this.control?.disabledChange` just in case a control is not provided? That should
 avoid a null reference error.

Reply:
Yes, absolutely. Fixed in last commit!